CODE_OF_CONDUCT.md

```diff
@@ -3,33 +3,33 @@
 ## Our Pledge
```

Comment:
do we want to move this into docs repo with a single reference for all other repos? docs.textile.io/terms/... is where others exist

Reply:
yeh that's probably a good idea! i just copied the updates from go-threads actually, but we might as well just link to a single source of truth in all repos and be done with it.